src/Components/Attendee.js

```diff
@@ -1,3 +1,31 @@
-export default function Attendee() {
-  return;
-}
+
+export default function Attendee({attendee,events,setEvents,event}) {
+
+  function updateEventAttendance(eventId, attendeeId) {
```

Comment:
It's more common to see helper functions like this defined in the same component as the state they update